ui/src/app/settings/actions/general/general.js

```diff
@@ -0,0 +1,14 @@
+const general = {};
+
+general.fetch = () => {
```

Comment:
This should probably be called `general.fetchOsInfo`

Reply:
Done